ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -3065,6 +3065,38 @@ def test_torch_instance_bitwise_or(
         on_device=on_device,
     )
 
+# bitwise_or_
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="torch.tensor",
+    method_name="bitwise_or_",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("integer"),
```

Comment:
@AnnaTz, Thanks for the feedback I've fixed the issues could u review this. Thanks !!! 😄

Reply:
@AmeenUrRehman The decorator is supposed to be added to the function itself, not its test. Also, the arguments you have passed to the decorator will make the function only support "bfloat16" which is not what we want here.

Reply:
@AnnaTz, Hi sorry it's my bad luck for that but are you sure that the decorator is needed here in the function because the test cases throw errors every time?

Reply:
@AnnaTz Hey could u help me in this.

Reply:
Are you still getting errors after removing the decorator from the test and adding it to the function as I suggested?

Reply:
Yes, basically all errors have the same cause of supporting 'integers' only. See I'm committing the code again before the decorator and u can check basically it is really needed here.